clusterloader2/pkg/execservice/exec_service.go

```diff
@@ -135,6 +135,26 @@ func RunCommand(cmd string) (string, error) {
 	return stdout.String(), nil
 }
 
+// PingNodePort ping hostIP:nodePort on a pod in cluster.
+func PingNodePort(nodePort int32) (string, error) {
```

Comment:
Seems like `PingNodePort` is very similar to `RunCommand`.
Maybe we can extract common code or call one function from the other ?

Reply:
updated!